frontend/craco.config.js

```diff
@@ -14,4 +14,19 @@ module.exports = {
     ],
     plugins: [],
   },
+  webpack: {
```

Comment:
Added to be able to import .obj|.drc|.gltf files. In the end we might not need this because they will be fetched, not imported